src/webui/www/public/newlabel.html

```diff
@@ -7,6 +7,17 @@
     <script type="text/javascript" src="scripts/mootools-1.2-core-yc.js" charset="utf-8"></script>
     <script type="text/javascript" src="scripts/mootools-1.2-more.js" charset="utf-8"></script>
     <script type="text/javascript">
+        var newLabelKeyboardEvents = new Keyboard({
```

Comment:
> This way is new for me and looks more complex than the old way

@ngosang ```Keyboard``` is the class of [MooTools](http://mootools.net/) library.

P.S. "Old way" doesn't work properly on key kombinations such as ```ctrl+enter```, ```alt+enter```, etc.